ui/scss/init/_gui.scss

```diff
@@ -18,6 +18,15 @@ html {
 
   color: var(--color-text);
   background-color: var(--color-background);
+  font-size: 16px;
```

Comment:
This most likely has a lot of consequences as it was previously 14px. At a minimum, you can see all buttons get larger, as well as subheaders on file list items.